Lib/test/test_annotationlib.py

```diff
@@ -42,11 +42,14 @@ def test_enum(self):
         self.assertEqual(Format.VALUE.value, 1)
         self.assertEqual(Format.VALUE, 1)
 
-        self.assertEqual(Format.FORWARDREF.value, 2)
-        self.assertEqual(Format.FORWARDREF, 2)
+        self.assertEqual(Format.VALUE_WITH_FAKE_GLOBALS.value, 2)
```

Comment:
See related comments, I don't think the fact that `STRING` format is the int object 2 is or should be part of the interface.  I don't want people hard-coding it.